SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -414,10 +444,34 @@ void USpatialSender::SendRPC(TSharedRef<FPendingRPCParams> Params)
 		// Target object was destroyed before the RPC could be (re)sent
 		return;
 	}
-
 	UObject* TargetObject = Params->TargetObject.Get();
+
+	if (USpatialActorChannel* Channel = NetDriver->GetOrCreateSpatialActorChannel(TargetObject))
+	{
+		if (Channel->bCreatingNewEntity)
+		{
+			if (Params->Function->HasAnyFunctionFlags(FUNC_NetMulticast))
+			{
+				// TODO: UNR-1437 - Add Support for Multicast RPCs on Entity Creation
+				UE_LOG(LogSpatialSender, Warning, TEXT("Multicast RPC on Entity Creation is not supported"));
```

Comment:
Can this log be more clear? Such as: NetMulticast RPC triggered on Actor too close to initial creation.

Reply:
And then print out the name of the NetMulticast function and the name of the TargetObject it was triggered on

Reply:
Changed to suggested log and added TargetObject and function names.